fr-index.html

```diff
@@ -0,0 +1,56 @@
+---
+title: "Jamulus ‒ Jouez de la musique en ligne. Avec vos amis. Gratuitement."
+lang: "fr"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Musique"
+ldjSlogan: "Jouez de la musique en ligne. Avec vos amis. Gratuitement."
+ldjCreator: "Volker Fischer et divers contributeurs"
+metadescription: "Jamulus est un logiciel open-source qui permet aux musiciens de jouer de la musique, de répéter ou simplement de faire des « Jams » en temps réel via Internet."
+mAltProgIcon: "Icône de Jamulus"
+mTSlogan: "Jouez de la musique en ligne. Avec vos amis. Gratuitement."
+mTGetStartedNow: "Commencer maintenant !"
+mHTMLDownloadNow: 'Advancé : <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">télécharger directement</a>'
+---
+<div class="fx-row fx-row-center-xs" id="firstrow">
+  <div class="fx-col-100-xs">
+    <div itemprop="abstract">
+      <h2>Qu'est-ce que c'est Jamulus ?</h2>
+      Jamulus est un logiciel pour jouer, répéter ou juste « Jammer » avec vos amis, votre groupe ou n'importe qui rencontré en ligne. Utilisez votre ordinateur Windows, macOS ou GNU/Linux pour vous connecter aux serveurs de Jamulus dans le monde gratuitement avec votre connexion Internet standard. Ou hébergez votre propre serveur privé. Jamulus est adapté pour la haute qualité, le son en faible latence et vous permet de jouer ensemble facilement à distance et dans le temps. Regardez <a href="wiki/Demos" target="_blank" rel="noreferrer">comment les gens l'utilisent</a> !
+    </div>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">
+      <img alt="Bannière de Jamulus. Redirige vers la page « Pour commencer »" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
+    </a>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs">
+  <div class="fx-col-100-xs fx-col-50-l">
+     <h2>Jamulus partout dans le monde</h2>
+    De partout dans le monde, Jamulus permet à d'immenses chœurs de chanter, à des groupes de rock de jouer fort, et réunit des musiciens folkloriques et classiques.
+    Il est utilisé pour les cours de musique à distance des écoles et universités, en privé et en public. Même uniquement pour la parole - tout est en temps réel via Internet, comme si vous y êtiez en personne.
+    <h2>Besoin d'aide ?</h2>
+    <p>
+      Lisez la <a href="wiki/" target="_blank" rel="noreferrer">documentation</a>. Regardez bien la section <a href="wiki/Client-Troubleshooting" target="_blank" rel="noreferrer">« Problèmes potentiels »</a>!
+      Vous pouvez aussi demander de l'aide dans le <a href="https://sourceforge.net/p/llcon/discussion/" target="_blank" rel="noreferrer">forum</a>.
+    </p>
+  </div>
+  <div class="fx-col-100-xs fx-col-50-l">
+    <h2>Jamulus est open-source</h2>
```

Comment:
@gegeweb I propose to use your changes here.

Reply:
@Saimyx @trebmuh 
In my fist MR for the title it was
```html
<h2>Jamulus est libre&nbsp;!</h2>
```
And in the parapgraph
```html
		<p>Jamulus est un <a href="https://www.gnu.org/philosophy/free-sw.fr.html" title="Qu'est-ce que le logiciel libre ?" target="_blank" rel="noreferrer">logiciel libre</a> distribué sous la licence publique générale GNU (<a href="https://www.gnu.org/licenses/gpl-2.0.fr.html"
				target="_blank" rel="noreferrer">GPL</a>).<br/> Pas besoin d'être développeur ou d'avoir des compétences techniques pour contribuer. Quelle que soit votre compétence vous pouvez aider&nbsp;!<br/> Une idée&nbsp? Une fonctionnalité manque&nbsp;? Vous
			avez trouvé un bogue&nbsp;? Vous pouvez traduire le logiciel, la documentation ou le site Internet dans votre langue&nbsp;?<br/> Chechez le projet sur <a href="https://sourceforge.net/projects/llcon/" target="_blank" rel="noreferrer">SourceForge</a>			ansi que le <a href="https://github.com/corrados/jamulus" target="_blank" rel="noreferrer">code source sur Github</a> et proposez votre contribution&nbsp;!</p>
		<p>
```
« _Pas besoin d'être développeur ou d'avoir des compétences techniques pour contribuer. Quelle que soit votre compétence vous pouvez aider_ » is not in the original English text.
It is (in most of my practice ;)) an interpretation with some additional clarification or explanation rather than a literal translation.

Reply:
> ```
> <h2>Jamulus est libre&nbsp;!</h2>
> ```

With the link to th definition, it would be better

```html
<h2>Jamulus est <a href="https://www.gnu.org/philosophy/free-sw.fr.html" title="Qu'est-ce que le logiciel libre ?" target="_blank" rel="noreferrer">libre</a>&nbsp;!</h2>
```

Reply:
Yeah. We do not need a literal translation but keep close. I'd leave it like that but open an issue to change it for all translations (this will then be included later on).

Reply:
> In order to keep translation healthy, I think we need to follow the French proverb "_ne pas mettre la charrue avant les boeufs_" which is litteraly translating to "_do not put the plough before the oxen_".

Take a look at the `change`branch… We can anticipate, we won't have to come back on this point.

>For what is concerning the current issue here, and as long as the original English text is "open source", I'd suggest to translate it to "source ouverte". It'll be times to change it when the original (English) text will be changed.

The bells should be tuned ;)
In another comment you say that "logiciel libre" can be used for "open source" in French because it makes sense.

Otherwise, I agree with @Saimyx , we can keep "open source" untranslated if we want use it with free like in the comming english version:
_Jamulus est un logiciel libre et « Open Source »_

Reply:
I commited "Jamulus est un logiciel open-source" because the original text will be refer to "libre" before translations